aeternity/ae_mdw
# fix: aex9 presence check demands mnesia ctx

## What

Adds mnesia context (dirty read) for aex9 presence check (`aex9_presence_exists/2` call).

## Why

After async tasks this function reads in existing mnesia context.

## Validation steps

1. Rename `priv/migrations/20210921123001_index_aex9_account_presence.ex` to `priv/migrations/20210921123002_index_aex9_account_presence.ex`
2. Run the Mdw and check that the migration has run again (it takes long time for 2 contracts to complete the balance check). This might take 5 minutes.
